Monolog/Handler/IgnoreDeprecationHandler.php

```diff
@@ -9,17 +9,13 @@
 
 class IgnoreDeprecationHandler extends AbstractHandler
 {
-    public function __construct($level = Logger::DEBUG)
+    public function __construct(int|string $level = Logger::DEBUG)
     {
         parent::__construct($level, false);
     }
 
     public function handle(array $record): bool
     {
-        if (!isset($record['channel'], $record['message']) || $record['channel'] !== 'php') {
```

Comment:
why don't you need to check isset first?

Reply:
Because src/Monolog/Handler/HandlerInterface.php:57

@phpstan-param Record $record
@phpstan-type Record array{message: string, context: mixed[], level: Level, level_name: LevelName, channel: string, datetime: \DateTimeImmutable, extra: mixed[]}

Monolog set Record with all keys

src/Monolog/Logger.php:305

$record = [
                    'message' => $message,
                    'context' => $context,
                    'level' => $level,
                    'level_name' => $levelName,
                    'channel' => $this->name,
                    'datetime' => new DateTimeImmutable($this->microsecondTimestamps, $this->timezone),
                    'extra' => [],
                ];